assets/sass/04-elements/links.scss

```diff
@@ -16,8 +16,11 @@ a:hover {
 }
 
 .site a:focus {
-	outline: 2px solid var(--wp--style--color--link, var(--global--color-primary));
-	text-decoration: none;
+
+	&:not(.skip-link) {
```

Comment:
This would increase the specificity of all `a:focus` styles...
Could we instead add an override for the skip-link?